Content.Shared/_Impstation/Tools/Components/SolutionFuelDrainComponent.cs

```diff
@@ -0,0 +1,17 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Chemistry.Components;
+
+/// <summary>
+///     Denotes the solution that can be easily removed through any reagent container.
+///     Think pouring this or draining from a water tank.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class SolutionFuelDrainComponent : Component
+{
+    /// <summary>
+    /// Solution name that can be drained.
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public string Solution = "default";
```

Comment:
probably want to make this a required datafield?
```suggestion
    public string Solution = default!;
```